cmd/captplanet/run.go

```diff
@@ -100,6 +101,12 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		}(i, storagenode)
 	}
 
+	// start miniredis for the repair process
+	go func() {
+		m := miniredis.NewMiniRedis()
+		m.StartAddr(":6379")
```

Comment:
Use non-default redis-port, people often already have redis running on this port.